### PR TITLE
CONFIG: SETI-4110 Bump ruby version to 2.6

### PR DESCRIPTION
**Pull request by caohhung**
Use feature branch `puppet6` if needed

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: specs/serverspec-core.spec [hunk 1]
```diff
@@ -14,21 +16,29 @@ Source0:          %{name}.tar.gz
...
 
 %build
+source %{_scl_prefix}/enable
 bundle install --standalone --binstubs --without=development
 
 %install
 rm -fr $RPM_BUILD_ROOT
 mkdir -p $RPM_BUILD_ROOT%{install_dir}
 cp -a * .bundle $RPM_BUILD_ROOT%{install_dir}
+cp -a /opt/rh/rh-ruby26/root/usr/share/gems/* $RPM_BUILD_ROOT%{install_dir}/bundle/ruby/
```
Comment (chiendm) [inline specs/serverspec-core.spec:38]: Not sure if copy existing gems on the current build machines are reasonable.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by chiendm** [comment]
LGTM

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/serverspec-core-merge-compliance-check/7/ : SUCCESS in 16s

**Pull request merged**